jovian/utils/commit.py

```diff
@@ -147,6 +149,8 @@ def commit(message=None,
         _attach_files(outputs, slug, version, output=True)
         _attach_records(slug, version)
 
+        set_colab_or_kaggle_project(project)
```

Comment:
seems like this can be done in one place independent of colab/kaggle.

Reply:
Think it'll be better to keep it dependent on Colab/kaggle. Since local jupyter commits have a different mechanisim to cache project with slugs in `_parse_project`

Reply:
Let me know, whether to keep it dependent or independent